src/test/java/features/Login.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Login
+  This feature deals with the login functionality of the application
+
+  Scenario: Login with correct username and password
+    Given I navigate to the login page
+    When I enter the following for email address
+      | username |
+      | calendartesting12358 |
+    And I click on Next button
+    Then I should see the password field
+    When I enter the following for password
+      | password |
+      | _ZD8:t9Q |
+    And I click on Next button
+    Then I should see the Calendar button
+    When I click on my picture
+    And I click on Sign Out
+    Then I should see the password field
+
+
+  Scenario: Login with invalid username
+    Given I navigate to the login page
+    When I enter the following for email address
+      | username |
+      | test |
+    And I click on Next button
+    Then I should see the Next button
```

Comment:
You should add an extra check for this Scenario:
And I should see an error message
or
And I should see the email field

Because the next button is the same in the next layer and you do not know which next button you see.